zebra/zebra_vty.c

```diff
@@ -940,6 +972,7 @@ DEFPY(ip_route_vrf,
 	  tag (1-4294967295)                           \
 	  |(1-255)$distance                            \
 	  |label WORD                                  \
+	  |table (1-4294967295)                        \
```

Comment:
Please remove table from vrf route command.

Reply:
this parameter will be handled only in the case with vrf backend = VRF netns